save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -197,6 +192,27 @@ class ProjectController(
             ResponseEntity.ok(projectStatus.message)
         }
 
+    /**
+     * @param project
+     * @param authentication
+     * @return response
+     */
+    @PostMapping("/delete")
```

Comment:
why not `DeleteMapping` here?

Reply:
The point is that `WebTestServer` with `.delete()` method doesn't support `.body()` and the idea was to pass `Project` with a request, but now we decided to pass only `organizationName` and `projectName` instead.

Reply:
So? you decided to change parameters and will change Mapping?

Reply:
Yes, will use DELETE method instead of POST